### PR TITLE
Run acceptance tests on Debian 8

### PR DESCRIPTION
**Pull request by alexjfisher**

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like it fails on the redis service

**Comment by alexjfisher** [comment]
> Looks like it fails on the redis service

Fixed

File: spec/acceptance/spiped_spec.rb [hunk 1]
```diff
@@ -41,10 +41,15 @@
...
+      redis_service = if fact('operatingsystemmajrelease') == '8'
+                        'redis-server'
+                      else
+                        'redis'
+                      end
       describe command('sed -i "s/^bind.*/bind 127.0.0.1/g" /etc/redis/redis.conf') do
         its(:exit_status) { is_expected.to eq 0 }
       end
-      describe command('systemctl start redis') do
+      describe command("systemctl start #{redis_service}") do
```
Comment (ekohl) [inline spec/acceptance/spiped_spec.rb]: Should this be `describe service(redis_service) { it { is_expected.to be_running } }` to ensure it's running?

File: spec/acceptance/spiped_spec.rb [hunk 2]
```diff
@@ -41,10 +41,15 @@
 
   describe 'redis over tunnel' do
     if fact('hostname') == 'spipedserver'
+      redis_service = if fact('operatingsystemmajrelease') == '8'
```
Comment (ekohl) [inline spec/acceptance/spiped_spec.rb]: This will probably break on EL8 once it's out

Comment (alexjfisher) [reply spec/acceptance/spiped_spec.rb]: ah yes.  It's only in the acceptance test code, but will need adjusting when eventually centos 8 is out and we want to test on that too.

**Pull request merged**